server/www/classes/view/logs/LogConsole.js

```diff
@@ -0,0 +1,149 @@
+var logConsoleInstance = null; 
+var lastLogHoveredTimestamp = 0; // the most recent timestamp at wich a log was hovered
+var logIndexToHighlight = -1; // the index of the log to be highlited
+// Log Console is of type singleton, it should never be instanciated with its own constructor but with getInstance()
+class LogConsole
```

Comment:
It also could have been implemented as a vue.js component, but at least it does the job in a much better way than before!